src/apps/content/templates/account/signup.html

```diff
@@ -2,7 +2,9 @@
 
 {% load i18n %}
 
-{% block head_title %}{% trans "Signup" %}{% endblock head_title %}
```

Comment:
hmm,
I love head_title more

what do you think?

Reply:
my suggestion is to use {% block title %} for both <title> tag and <h1> tag. 
for instance, from product_list.html i pass {% block title %} Product List {% endblock title %}.
this block is used for <h1> tag in base_product_list.html and as <title> in base.html

my suggestion is that title of page and h1 tag should be always the same. SEO approves this solution, i guess)

Reply:
Ok, approved